base-notebook/Dockerfile

```diff
@@ -126,16 +126,19 @@ RUN set -x && \
             fix-permissions "/home/${NB_USER}"; \
     fi;
 
+ARG NOTEBOOK_VERSION=default
+ARG JUPYTERHUB_VERSION=default
+ARG JUPYTERLAB_VERSION=default
```

Comment:
If we change default values to empty strings, then the check is not needed and we can just write
```
mamba install --quiet --yes notebook="${NOTEBOOK_VERSION}"
```

And if someone specified version, they will get it and if not - they will get default version.